src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -430,19 +431,31 @@
     type: command
     short-summary: Configure the Azure VM diagnostics extension.
     examples:
-        - name: Set up default diagnostics on a Linux VM.
+        - name: Set up default diagnostics on a Linux VM for Azure Portal VM metrics graphs and syslog collection.
           text: >
+            # Set the following 3 parameters correctly first.\n\r
+            my_resource_group=<Your Azure resource group name containing your Linux VM and the storage account>\n\r
```

Comment:
Nit, since we are in `azure cli` already, the `Your Azure` can be omitted,  I think